src/proofOfConcept.java

```diff
@@ -0,0 +1,79 @@
+import components.list.List;
+
+public class FantasyFootball<String> {
+
+    /**
+     * Elements included in {@code this}.
+     */
+    private List<String> football;
```

Comment:
I don't usually love index based data structures for this unless you specify what each index means for the user. Maybe you can store Player objects instead of strings and then provide a variety of useful methods for retrieving players by position, points, etc.